src/TodoEdit.js

```diff
@@ -0,0 +1,71 @@
+import {useState} from 'react';
+import {TodosForm} from './TodosForm';
+import {RiCloseCircleLine} from 'react-icons/ri';
+import {TiEdit} from 'react-icons/ti';
+import styled from 'styled-components';
+
+function TodoEdit({todos, completeTodo, removeTodo, updateTodo}) {
+  const [edit, setEdit] = useState({
+    id: null,
+    value: '',
+  });
+  const submitUpdate = value => {
+    updateTodo(edit.id, value);
+    setEdit({
+      id: null,
+      value: '',
+    });
+  };
+
+  if (edit.id) {
+    return <TodosForm edit={edit} onSubmit={submitUpdate} />;
+  }
+
+  return todos.map((todo, index) => (
+    <StyledComplete className={todo.isComplete ? 'todo-row complete' : 'todo-row'} key={index}>
+      <aside key={todo.id} onClick={() => completeTodo(todo.id)}>
+        {todo.text}
+      </aside>
```

Comment:
If you want the user to click on an element, it should always be a button (A11y).